src/components/about/about.tsx

```diff
@@ -0,0 +1,8 @@
+export default function About() {
+  return (
+    <div className="background-image center-text">
+      <h1>About page</h1>
```

Comment:
please use h2 here. h1 can be single and this is Game Store title in the header

Reply:
Done